lib/util.js

```diff
@@ -26,7 +26,7 @@ function getArg(aArgs, aName, aDefaultValue) {
 }
 exports.getArg = getArg;
 
-var urlRegexp = /^(?:([\w+\-.]+):)?\/\/(?:(\w+:\w+)@)?([\w.-]*)(?::(\d+))?(\S*)$/;
+var urlRegexp = /^(?:([\w+\-.]+):)?\/\/(?:(\w+:\w+)@)?([\w.-]*)(?::(\d+))?(.*)$/;
```

Comment:
While reviewing, I realized that in `isAbsolute`, you should use `urlRegexp.test(aPath)` instead of `aPath.match(urlRegexp)` because you don't need the matches, so it should be faster. You can do it in this patch or in another patch if you want.

Reply:
I made the change.

Reply:
Actually, this change causes many regressions.

Reply:
... because I wrote it incorrectly.